widget/ui/src/components/ExpandedRoute/ExpandedRoute.tsx

```diff
@@ -0,0 +1,151 @@
+import type { PropTypes } from './ExpandedRoute.types';
+
+import React from 'react';
+import { ErrorIcon, WarningIcon } from 'src/icons';
```

Comment:
When run storybook (yarn dev:ui), 
we have this error:
Error: Cannot find module 'src/icons'